### PR TITLE
docs: improve external approval doc

### PR DESCRIPTION
**Pull request by RainbowDashy**

### CODE REVIEW

**Review by d-bytebase — CHANGES_REQUESTED** [review]
png -> webp

**Review by tianzhou — CHANGES_REQUESTED** [review]
CleanShot 2022-11-17 at 10 09 48

TOC not right

We should split into 2 sections, one is Feishu setup, one is Bytebase setup

File: content/docs/administration/webhook-integration/external-approval.md [hunk 1]
```diff
@@ -4,21 +4,51 @@ title: External Approval
...
+Get the **App ID** and **App Secret** in the General info section of Credentials & Basic Info page. You can add [Bytebase icon](https://www.bytebase.com/brand) on the same page.
+![feishu app credential page](/static/docs/external-approval-feishu-credential.png)
+
+## Step 2 - Setup
 
 1. Go to Bytebase Settings, click on IM Integration.
 1. Fill in Application ID and Secret from the previous section. Click on Enable and Create button.
-1. It's all set. **Remember**, the registered Bytebase account must have an identical email at Feishu to receive Approval requests.
\ No newline at end of file
+1. It's all set.
```
Comment (candy2255) [inline content/docs/administration/webhook-integration/external-approval.md:51]: How about deleting "It‘s all set."?

**Comment by RainbowDashy** [comment]
PTAL

**Review by candy2255 — APPROVED** [review]
LGTM

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**